package-installer/src/main/java/com/vmware/pscoe/iac/installer/Installer.java

```diff
@@ -764,8 +764,8 @@ private static void loadInputInteractiveMode(Input input) {
             userInput(input, Option.VRANG_IMPORT, "Import vRA8 packages?", true);
           
         }
-    
-		if (input.anyTrue(Option.CS_IMPORT)) {
+
+        if (input.anyTrue(Option.VRANG_IMPORT, Option.CS_IMPORT)) {
```

Comment:
Lets take a look at: https://jira.pscoe.vmware.com/browse/IAC-584 first. 

I think we may need to extract readVrangProperties(input); from this if else and leave it as is